### PR TITLE
WorlItem-10200181:Remove the "last updated" column in per-table grid …

### PR DESCRIPTION
**Pull request by v-haripal**
…on the usage tab

WorlItem-10200181:Remove the "last updated" column in per-table grid on the usage tab

## PR Checklist

* [ ] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.

### If adding or updating templates:
* [ ] post a screenshot of templates and/or gallery changes
* [ ] ensure your template has a corresponding gallery entry in the gallery folder
* [ ] If you are adding a new template, add your team and template/gallery file(s) to the CODEOWNERS file. CODEOWNERS entries should be teams, not individuals
* [ ] ensure all steps have meaningful names
* [ ] ensure all parameters and grid columns have display names set so they can be localized
* [ ] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [ ] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [ ] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [ ] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

### CODE REVIEW

File: Workbooks/LogAnalytics Workspace/Usage/LA Workspace Insights Usage.workbook [hunk 1]
```diff
@@ -403,7 +403,7 @@
                           }
                         },
                         {
-                          "columnMatch": "-- Group By --",
+                          "columnMatch": "$gen_group",
```
Comment (noakup) [inline Workbooks/LogAnalytics Workspace/Usage/LA Workspace Insights Usage.workbook:406]: I don't know what this change would affect, probably better to keep the "-- Group By --"

File: Workbooks/LogAnalytics Workspace/Usage/LA Workspace Insights Usage.workbook [hunk 2]
```diff
@@ -419,9 +419,6 @@
                         "expandTopLevel": true
                       },
                       "labelSettings": [
-                        {
-                          "columnId": "Solution"
-                        },
```
Comment (noakup) [inline Workbooks/LogAnalytics Workspace/Usage/LA Workspace Insights Usage.workbook:424]: was this removed intentionally?

File: Workbooks/LogAnalytics Workspace/Usage/LA Workspace Insights Usage.workbook [hunk 3]
```diff
@@ -433,13 +430,7 @@
...
                           "columnId": "IngestionPercent",
                           "label": "Ingestion Percentage"
-                        },
-                        {
-                          "columnId": "TimeSinceLastUpdate",
-                          "label": "Usage Stats Updated"
-                        },
-                        {
-                          "columnId": "Billable"
+
```
Comment (noakup) [inline Workbooks/LogAnalytics Workspace/Usage/LA Workspace Insights Usage.workbook:433]: Was the Billable column removed intentionally?

File: Workbooks/LogAnalytics Workspace/Usage/LA Workspace Insights Usage.workbook (hunk 1 above)
Comment (v-haripal) [reply Workbooks/LogAnalytics Workspace/Usage/LA Workspace Insights Usage.workbook]: added: -- Group By --

File: Workbooks/LogAnalytics Workspace/Usage/LA Workspace Insights Usage.workbook (hunk 2 above)
Comment (v-haripal) [reply Workbooks/LogAnalytics Workspace/Usage/LA Workspace Insights Usage.workbook:424]: Added: column-Solution

File: Workbooks/LogAnalytics Workspace/Usage/LA Workspace Insights Usage.workbook (hunk 3 above)
Comment (v-haripal) [reply Workbooks/LogAnalytics Workspace/Usage/LA Workspace Insights Usage.workbook:436]: Added: columnId- Billable

**Review by noakup — APPROVED** [review]

**Review by noakup — APPROVED** [review]

**Pull request merged**